frontend/src/store/useTaskStore.js

```
@@ -0,0 +1,103 @@
+},
+
+deleteTask: async (id) => {
+  set({ isDeletingTask: true, taskError: null });
+  try {
+    await axiosInstance.delete(`/tasks/${id}`);
+    set((state) => ({
+      tasks: state.tasks.filter((task) => task._id !== id),
+    }));
+  } catch (error) {
+    console.error("Error deleting task:", error);
+    set({ taskError: "Failed to delete task: " + error.message });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)